content/blog/2018/delayering-istio/delayering-istio.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Delayering Istio with AppSwitch
+description: Automatic application onboarding and latency optimizations by removing unnecessary layers with AppSwitch
```

Comment:
I would not call these layers unnecessary until AppSwitch can be used as a 100% replacement for Envoy for all language runtimes.
Please rephrase as "Enhancing performance of certain latency-sensitive applications using AppSwitch in-process load-balancing

Reply:
The layers are unnecessary for the cases / reasons discussed but I'll remove reference to it.